### ISSUE TITLE
ebay-icon: DS4 icons are not part of the symbols library

### ISSUE THREAD

**Issue by seangates** [eBay/ebayui-core#315]
# Bug Report

## Description
Currently, the only icons in the `ebay-icon/symbols` directory are for DS6. We need to be able to support different design systems' icons. One solution is to possibly use the `skin-ds#` Lasso flag which gives us this information. I don't believe we need Skin to be tied to the components, though, which puts us in a tough spot.

Please let me know your thoughts on how to proceed.

## Workaround
None.

**Comment by sendlo** [comment]
The only other options I can think of is to have different icon names for DS4 icons, or set some additional flag on the component.  But in both cases that means the client has to update the component when moving version of skin, which is a non-starter.

I think its OK to base this off of the skin lasso flag, with a default to the latest style, since it represents a design change anyway.  

Does the renderer have access to the lasso flags?  I've never had to do that.

**Comment by DylanPiercey** [comment]
What's the issue with using lasso flags for this?

The way I see it we would do something like:

```
ebay-icon/
  symbols/
    add/
      browser.json # Remaps between ds4 and ds6
      ds4.marko
      ds6.marko
```

**Comment by seangates** [comment]
And in `browser.json` we do the check, then send the appropriate template to the bundler?

**Comment by DylanPiercey** [comment]
Yeah, the browser json for each icon folder would look like:

```
{
    "dependencies": [
        {
            "if-not-flag": "skin-ds6",
            "path": "./ds4.marko"
        },
        {
            "if-flag": "skin-ds6",
            "path": "./ds6.marko"
        }
    ]
}
```

**Comment by seangates** [comment]
My gut just tells me "don't tie yourself to the horse" ... a.k.a. decouple from the design system. And this ties us to the design systems we're using.

It's just hard not to do it since the icon names are 1:1 with the design system names, essentially tying us to certain stamped icon IDs. I guess that's an okay tradeoff in this scenario.

**Comment by sendlo** [comment]
yep.  I see your concern, but in this specific instance I think its the best option.

**Comment by seangates** [comment]
Let's prioritize this work as part of 1.0.0.
